source/agora/consensus/Ledger.d

```diff
@@ -757,11 +763,11 @@ public class ValidatingLedger : NodeLedger
 
     ***************************************************************************/
 
-    public Hash[] getCandidateTransactions (in Height height, ulong max_txs,
-        scope UTXOFinder utxo_finder) @safe
+    public Hash[] getCandidateTransactions (in Height height, scope UTXOFinder utxo_finder) @safe
     {
         Hash[] result;
         Amount tot_fee, tot_data_fee;
+        size_t size_budget = this.params.MaxTxSetSize * 1024; // cant overflow
```

Comment:
```suggestion
        size_t size_limit = this.params.MaxTxSetSize * 1024; // cant overflow
```

Reply:
I applied previous suggestions but since we keep subtracting from this, it actually represents the budget we have left. So it seem like a better name.